### DISCUSSION TITLE
How long should it take for Deno Deploy to update?

### DISCUSSION THREAD

**Discussion by jtlapp** [freshframework/fresh#905] [Q&A] [answered]
I'm using Fresh with Deno Deploy and am wondering how long it takes for the deployment to update after a git push. I hadn't thought about this possible consequence of using Deno Deploy before deciding on it. It's certainly not quick.

The answer is important for Fresh, because if I'm not using Deno Deploy, I'll need to put it behind my existing NGINX/Node installation and therefore port to something other than Fresh.

UPDATE: One hour later, the site still hasn't updated. This isn't going to work for me. Fresh was fun, but I'm porting to next.js.

**Comment by xyntechx** [comment]
That's strange. I've made 2 Fresh projects so far and deployed them to Deno Deploy with no issues whatsoever. Could it be that you had unaddressed errors, like linting with `deno fmt` for example? Maybe I can help if you could send me the link to your repo!

P.S. I'm actually not sure whether running `deno fmt` before deployment is necessary, but you should give it a try.

**Comment by marvinhagemeister** [comment, marked as answer]
Deployments should only take a couple of seconds. If you encounter an issue with an unexpectedly slow deployment please file an issue here https://github.com/denoland/deploy_feedback
